Topics: rename SIGHASH_NOINPUT to SIGHASH_ANYPREVOUT (#627)
bitcoinops/bitcoinops.github.io: _topics/en/sighash_anyprevout.md

  - Scripts and Addresses

aliases:
  - SIGHASH_NOINPUT

## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
excerpt: >

>   **SIGHASH\_ANYPREVOUT**, previously known also as SIGHASH\_NOINPUT, is a proposal for a

Reviewer: ```suggestion
  **SIGHASH\_ANYPREVOUT**, an updated version of SIGHASH\_NOINPUT, is a proposal for a
```
